string keyword
string answer
string update_time
uint8 action
uint8 action_mode